src/graph/network/common.h

```diff
@@ -0,0 +1,132 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file common.h
+ * \brief This file provide basic facilities
+ * to make programming convenient.
+ */
+#ifndef DGL_GRAPH_NETWORK_COMMON_H_
+#define DGL_GRAPH_NETWORK_COMMON_H_
+
+#include <dmlc/logging.h>
+
+#include <set>
+#include <string>
+#include <vector>
+
+namespace dgl {
+namespace network {
+
+//------------------------------------------------------------------------------
+// Subdivide string |full| into substrings according to delimitors
+// given in |delim|.  |delim| should pointing to a string including
+// one or more characters.  Each character is considerred a possible
+// delimitor. For example:
+//
+//   vector<string> substrings;
+//   SplitStringUsing("apple orange\tbanana", "\t ", &substrings);
+//
+// results in three substrings:
+//
+//   substrings.size() == 3
+//   substrings[0] == "apple"
+//   substrings[1] == "orange"
+//   substrings[2] == "banana"
+//------------------------------------------------------------------------------
+
+void SplitStringUsing(const std::string& full,
+                      const char* delim,
+                      std::vector<std::string>* result);
```

Comment:
Check out functions in [dmlc-core/strtonum.h](https://github.com/dmlc/dmlc-core/blob/master/include/dmlc/strtonum.h) to see whether there are sth useful.

Reply:
It seems that nothing we can borrow from this file.